Towny/src/main/java/com/palmergames/bukkit/towny/permissions/PermissionNodes.java

```diff
@@ -249,6 +249,7 @@ public enum PermissionNodes {
 	TOWNY_COMMAND_RESIDENT_LIST("towny.command.resident.list"),
 	TOWNY_COMMAND_RESIDENT_TAX("towny.command.resident.tax"),
 	TOWNY_COMMAND_RESIDENT_PLOTLIST("towny.command.resident.plotlist"),
+	TOWNY_COMMAND_RESIDENT_OUTLAWLIST("towny.command.resident.outlawlist"),

```

Comment:
This new node should be
- [ ] added to the nomad group in the default townyperms.yml in the resources package.
- [ ] added to to the config migration system to be added to already existing townyperms.yml files.